### PR TITLE
Add rules_erlang 3.8.5

### PR DESCRIPTION
**Pull request by pjk25**
https://github.com/rabbitmq/rules_erlang/releases/tag/3.8.5

### CODE REVIEW

File: modules/rules_erlang/3.8.5/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,40 @@
+module(
+    name = "rules_erlang",
+    compatibility_level = 3,
+    version = "3.8.5",
+)
+
+bazel_dep(
+    name = "rules_pkg",
+    version = "0.5.1",
+)
```
Comment (fmeum) [inline modules/rules_erlang/3.8.5/MODULE.bazel:10]: Just a drive-by comment: Could this be a dev dependency or do you use it outside the release process?

**Review by fmeum — APPROVED** [review]

File: modules/rules_erlang/3.8.5/MODULE.bazel (hunk 1 above)
Comment (pjk25) [reply modules/rules_erlang/3.8.5/MODULE.bazel:10]: It's used `source_archive.bzl` for the `source_archive` macro, so I don't believe it's a dev dependency. But thanks for pointing it out. If I have a misunderstanding of how `dev_dependency` is supposed to be used, I'd be happy to learn more.

**Comment by pjk25** [comment]
@kormide The automatic PR based on the .bcr folder did not seem to work this time either. Did I miss an automation failure somewhere?

**Comment by kormide** [comment]
@pjk25 It looks like it couldn't find the GitHub app installation for `rabbitmq/bazel-central-registry`. Did you install the app to that repository?
